src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -91,6 +91,13 @@ public sealed class MatrixFactorizationTrainer : TrainerBase<MatrixFactorization
     {
         public sealed class Arguments
         {
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Loss function minimized for finding factor matrices. " +
+                "Two values are allowed, 0 or 12. The values \"0\" means traditional collaborative filtering problem with squared loss. " +
+                "The value \"12\" triggers one-class matrix factorization for implicit-feedback recommendation problem.")]
+            [TGUI(SuggestedSweeps = "0,12")]
+            [TlcModule.SweepableDiscreteParam("Fun", new object[] { 0, 12 })]
+            public int Fun = 0;
```

Comment:
Do you want to add test which will cover this?

Reply:
Added. Thanks.